src/main/steps/index.test.ts

```diff
@@ -161,18 +161,25 @@ describe('Steps', () => {
 
   describe('isConditionalOrderReadyToSubmit()', () => {
     it('returns false if nextStepUrl is /continue-with-your-application', () => {
-      const isApplicationReadyToSubmitBoolean = isConditionalOrderReadyToSubmit(CONTINUE_WITH_YOUR_APPLICATION);
+      const isApplicationReadyToSubmitBoolean = isConditionalOrderReadyToSubmit(
+        CONTINUE_WITH_YOUR_APPLICATION,
+        HOME_URL
+      );
```

Comment:
and this test

Reply:
Lara, so would the second argument be WITHDRAWING_YOUR_APPLICATION ? as only when the user chooses not to continue, they would then be directed to the Check your answer so far page.

Reply:
if you see the isConditionalOrderReadyToSubmit function the second parameter is 
`applicantApplyForConditionalOrder: string`
which is the applicantApplyForConditionalOrder. So when testing the function you need to give it an argument in this form (which in this case is either Yes or No)

Reply:
Ah ok, I see, thanks, yeah I updated the index test.